dist/ads.js

```diff
@@ -127,32 +135,47 @@ const parseSellersJSON = async (response) => {
             seller_count: 0,
           }
         },
-        passthrough_count: 0
+        passthrough_count: 0,
+        confidential_count: 0
       }
     };
 
     // Clean up file content
     result.seller_count = content.sellers.length;
 
     for (let seller of content.sellers) {
+      // Validating records
+      if (!seller.seller_type && !seller.id) {
```

Comment:
```suggestion
      const type = seller.seller_type.trim().toLowerCase(),

      // Validating records
      if (!SELLER_TYPES.includes(type) || !seller.seller_id) {
```

Both of these fields are required.